### PR TITLE
Closes Issue #372

### PR DESCRIPTION
**Pull request by andrewnguyen22**
General restructure of mainnet repo for clarity

### CODE REVIEW

File: service/.keep [hunk 1]
```diff
@@ -0,0 +1 @@
+// add economic knobs to this package
```
Comment (luyzdeleon) [inline service/.keep]: Wrong comment

File: session/.keep [hunk 2]
```diff
@@ -0,0 +1 @@
+// add economic knobs to this package
```
Comment (luyzdeleon) [inline session/.keep]: Wrong comment

File: tests/bdd/fixtures/nodeGenerator.go [hunk 3]
```diff
@@ -39,7 +37,7 @@ func createNode(nodeNumber int) common.NodeWorldState {
 	hasher.Write([]byte("node" + strconv.Itoa(i)))
 =======
 =======
-func createNode(nodeNumber int) core.NodeWorldState {
+func createNode(nodeNumber int) types.NodeWorldState{
```
Comment (luyzdeleon) [inline tests/bdd/fixtures/nodeGenerator.go]: Git diff characters in place

**Review by luyzdeleon — CHANGES_REQUESTED** [review]

**Comment by andrewnguyen22** [comment]
Changes made, ready to close pre-mainnet milestone

**Review by luyzdeleon — APPROVED** [review]

**Pull request merged**